docs/gettingstarted/index.md

```diff
@@ -6,18 +6,19 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded MDK to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain a Java 8 Development Kit (JDK) and a 64-bit Java Virtual Machine (JVM). Minecraft and MinecraftForge both compile against Java 8 and as such should be used for development. Using a 32-bit JVM will result in some problems when running the below gradle tasks. You can obtain one from [AdoptOpenJDK][jdk].
+2. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
```

Comment:
```suggestion
2. Obtain the Mod Development Kit (MDK) from Forge's [files][] site (Look for the MDK file type).
```
Should we leave out the parenthetical expression to look for the MDK file type? Since as of recently, only three download buttons are displayed for the Latest and Recommended versions: the Installer, the Changelog, and the MDK.

Reply:
Probably since it's already clear based on the original abbreviation.